internal/server/node/golden/get_property_values/contained_in_place.json

```diff
@@ -146,7 +146,7 @@
             "City"
           ],
           "dcid": "geoId/0627080",
-          "provenanceId": "dc/sm3m2w3"
+          "provenanceId": "dc/5n63hr1"
```

Comment:
Need to debug why this is...

Reply:
@Spaceenter need to debug this in Prophet.